workspaces/redhat-argocd/plugins/argocd/src/components/DeploymentLifeCycle/sidebar/resources/ResourcesTableRow.tsx

```diff
@@ -49,11 +40,15 @@ const useStyles = makeStyles(theme => ({
 export const ResourcesTableRow: React.FC<ResourcesTableRowProps> = ({
   uid,
   row,
-  createdAt,
 }) => {
   const classes = useStyles();
   const [open, setOpen] = React.useState(false);
 
+  const formattedTimestamp = useMemo(
+    () => moment.utc(row?.createTimestamp).local().format('MM/DD/YYYY hh:mm a'),
```

Comment:
It is possible that `row?.createTimestamp` is `undefined`, In that case moment is show the current time to the user which is incorrect, incase of undefined we should just show  as hypen `-` or leave it empty.

![image](https://github.com/user-attachments/assets/0fb28d42-5166-4754-a579-2b9867d290bb)